command/genesis/genesis.go

```diff
@@ -273,6 +273,14 @@ func setFlags(cmd *cobra.Command) {
 			defaultVoteProposalThreshold,
 			"number of vote tokens required in order for a voter to become a proposer",
 		)
+
+		cmd.Flags().StringVar(
+			&params.governorAdmin,
```

Comment:
Just asking: there cannot be multiple governor admins?

Reply:
Nope. Only one.